kbase-extension/static/kbase/js/common/pythonInterop.js

```diff
@@ -244,16 +251,53 @@ define([], () => {
         return pythonCode;
     }
 
+    /**
+     * Builds the call to run_app_bulk
```

Comment:
Maybe we should chat about this structure. I think this'll be the most flexible down the road for whoever else wants to run it, but it duplicates all of the common parameters. I also considered adding a separate "shared_params" dict to each app that gets run in bulk, and just put it on the kernel to do the duplication before running in EE2. But... I haven't done that yet.

Reply:
Yes, it might be worth thinking about / discussing -- there's also the matter of how to display the params in the job status table; we don't really want to repeat the shared params for each job in a batch. I'm also wondering about whether we store the input params in the frontend model object or in the kernel so that info is immediately accessible to the job status table and there doesn't need to be an extra call to ee2 for every job in the table.

Reply:
I think the kernel can hold it pretty easily. It might have a slightly higher memory footprint on the server, but it's easy enough to make a request for that information. I think at kernel startup when it fetches all the job info, it can probably fetch and cache all of that pretty quickly, then we can have the FE request it as needed.